### ISSUE TITLE
ARM VM

### ISSUE THREAD

**Issue by iltempe** [jordansissel/xdotool#284]
Is possible to compile tool as for ARM processor? Is there a Virtual Machine for this? Thanks

**Comment by madscientist42** [comment]
It's plausible if you use something other than the current Makefile build system (See #293...)

Once you have a CMake or Meson based build system, cross-compile becomes much, much easier without having Scratchbox2 or similar in hand.

**Comment by madscientist42** [comment]
(As an aside, I have built this to work on an ARM (More specifically a Tegra X2...) target and it works decently enough.)

**Comment by zcake3000** [comment]
Yes it can compiled for arm.There have some linux distro (such at arch arm,debian) have been already compiled at a package for arm
